Scripts/BrightSign_Scripts/bsext_init

```diff
@@ -14,6 +14,7 @@ if [[ $1 == "start" ]]; then
     start-stop-daemon --start \
         --pidfile $TRK_PIDFILE --make-pidfile \
         --background --exec $TRK_DAEMON
+    sleep 10
```

Comment:
Is 10s overkill? Could we reduce this

Reply:
I will leave the story open to investigate further, but merging this for now.